nyc/templates/search/indexes/nyc/bill_text.txt

```diff
@@ -14,5 +15,4 @@
 {% for t in object.topics %}
   {{t}}
 {% endfor %}
-{{ object.full_text|clean_html }}
-{{ object.ocr_full_text|clean_html }}
\ No newline at end of file
+{{ object.ocr_full_text|clean_plain_text|clean_html }}
```

Comment:
Is there ever an instance where Solr might index a bill that does not have a value for [`ocr_full_text`](https://github.com/datamade/django-councilmatic/blob/6f5bbe2a752a9f40c3683b3bedf11c77114576fe/councilmatic_core/management/commands/import_data.py#L920)? (I'd guess no, but just checking!)

Reply:
Do we have bills without plain text?

Reply:
Dunno!

Reply:
All right....we do have bills without ocr_full_text (i.e., plain_text). I am not sure how Haystack deals with that, but I'll look into it.

Reply:
Okay - I read about this @hancush. The `bill_text` template insures that Haystack indexes bills and makes them searchable, even when they do no have `ocr_full_text`: http://django-haystack.readthedocs.io/en/v2.4.1/best_practices.html#well-constructed-templates. Solr will search through all the fields defined in the custom template (so, if the bill does not have plain text, then Solr will still search its descriptions, actions, etc). I removed the `full text` field (which contains RTF) from this custom template because (1) we have all the same data in the `ocr_full_text`, and (2) we do not want to show search results in RTF, so it's best to just not index it in Solr.